### PR TITLE
[LCP] non-trusted event does not stop the metric

### PR DESCRIPTION
**Pull request by yoavweiss**
Tests https://github.com/WICG/largest-contentful-paint/pull/33

### CODE REVIEW

File: largest-contentful-paint/observe-after-untrusted-scroll.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  async_test(function (t) {
+    if (!window.LargestContentfulPaint) {
+      assert_unreached("LargestContentfulPaint is not implemented");
+    }
+    const beforeRender = performance.now();
+    const observer = new PerformanceObserver(
+      t.step_func_done(function(entryList) {
+        assert_equals(entryList.getEntries().length, 1);
+        const entry = entryList.getEntries()[0];
+        assert_equals(entry.entryType, 'largest-contentful-paint');
```
Comment (npm1) [inline largest-contentful-paint/observe-after-untrusted-scroll.html]: These checks are fine though the key is that the observer must get an entry, right?

File: largest-contentful-paint/observe-after-untrusted-scroll.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        assert_equals(entry.url, pathname);
+        assert_greater_than(entry.loadTime, beforeRender,
+          'The load timestamp should occur after script starts running.');
+        assert_less_than(entry.loadTime, entry.renderTime,
+          'The load timestamp should occur before the render timestamp.')
+        assert_equals(entry.element, document.getElementById('image_id'));
+      })
+    );
+    observer.observe({type: 'largest-contentful-paint', buffered: true});
+  }, 'Same-origin image is observable.');
```
Comment (npm1) [inline largest-contentful-paint/observe-after-untrusted-scroll.html]: Nit: update test description.

**Review by npm1 — APPROVED** [review]

File: largest-contentful-paint/observe-after-untrusted-scroll.html (hunk 1 above)
Comment (yoavweiss) [reply largest-contentful-paint/observe-after-untrusted-scroll.html]: Indeed!

File: largest-contentful-paint/observe-after-untrusted-scroll.html (hunk 2 above)
Comment (yoavweiss) [reply largest-contentful-paint/observe-after-untrusted-scroll.html]: Oops! updated

**Pull request merged**